Cargo.toml

```diff
@@ -11,12 +11,17 @@ tracing = "0.1"
 tracing-subscriber = "0.1"
 clap = "2.33"
 prost = "0.6"
-rusk-vm = { git = "ssh://git@github.com/dusk-network/rusk-vm.git", tag = "v0.1.2" }
-phoenix-abi = { git = "https://github.com/dusk-network/phoenix-abi" }
-kelvin = "0.11.2"
-bytehash = "0.2"
+# rusk-vm = { git = "ssh://git@github.com/dusk-network/rusk-vm.git", tag = "v0.1.2" }
```

Comment:
Just to be sure you don't forget - let's set this import to HTTPS once it is uncommented